src/viam/ur/module/ur_arm.cpp

```diff
@@ -498,7 +578,72 @@ ProtoStruct URArm::do_command(const ProtoStruct& command) {
     return resp;
 }
 
-void URArm::move_(std::shared_lock<std::shared_mutex> config_rlock, std::list<Eigen::VectorXd> waypoints, const std::string& unix_time) {
+void URArm::move_tool_space_(std::shared_lock<std::shared_mutex> config_rlock, pose p, const std::string& unix_time) {
+    auto our_config_rlock = std::move(config_rlock);
+
+    // Capture the current movement epoch, so we can later detect if another caller
+    // slipped in while we were planning.
+    auto current_move_epoch = current_state_->get_move_epoch();
+
+    VIAM_SDK_LOG(info) << "move tool space: start unix_time_ms " << unix_time << " p: " << p;
```

Comment:
Make sure you and @JohnN193 coordinate your merges re #117 so you don't re-introduce info level logging that should get reduced to debug.

Reply:
added suggestions for all the spots that should be debug logs